components/Modules/modules-content.tsx

```
@@ -388,13 +399,13 @@ export default function ModulesPageContent({
               <Suspense fallback={<Button disabled>Loading...</Button>}>
                 <ModuleOperations sessionId={sessionId} />
               </Suspense>
-            ) : (
+            ) : isSignedIn ? (
               <ResourceUploadButton
                 variant="outline"
                 moduleId={preselectedModuleId || undefined}
                 initialOpen={openResourceUpload}
               ></ResourceUploadButton>
```

Biome (1.9.4)

[error] JSX elements without children should be marked as self-closing. In JSX, it is valid for any element to be self-closing.

Unsafe fix: Use a SelfClosingElement instead


(lint/style/useSelfClosingElements)